CreeDictionary/search_quality/sample.csv

```diff
@@ -0,0 +1,64 @@
+English,Nêhiyawêwin 1,Nêhiyawêwin 2,Nêhiyawêwin 3,,,Poor result,Notes
+star,acâhkos,atâhk,,,,,
+see,wâpamêw,wâpiw,wâpahtam,,,kwêyask,"Ignore ""\[see"""
+eat,mîcisow,mîciw,mowêw,,,,
+fire,iskotêw,,,,,,
+happy birthday,miyo-tipiska,,,,,,
+thank you,ay-hay,hay-hay,,,,,
+dog,atim,,,,,,very irregular inflections
+horse,misatim,,,,,,
+'horse',misatim,,,,,,
+"""horse""",misatim,,,,,,
+beaverberry,amiskomin,amiskominak,,,,,
+beaverberries,amiskomin,amiskominak,,,,,
+beaver-berry,amiskomin,amiskominak,,,,,
+beaver-berries,amiskomin,amiskominak,,,,,
+beaver berry,amiskomin,amiskominak,,,,,
+beaver berries,amiskomin,amiskominak,,,,,
+berries,mînisa,mînis,,,,,
+berry,mînis,mînisa,,,,,
+berri,mînis,mînisa,,,,,
+stove pipe,okohtaskwahikan,,,,,,
+stove-pipe,okohtaskwahikan,,,,,,
+stovepipe,okohtaskwahikan,,,,,,
+in the area,cîkâhtaw,,,,,,
+bread,pahkwêsikan,,,,,,
+yellow hat,osâwastotin,,,,,,
+sleep,nipâw,,,,,,
+I love you,kisâkihitin,,,,,,
+this,ôma,awa,ôhi,,,,
+that,anima,,,,,,
+good morning,miyo-kîkisêpâyâw,,,,,,
+me,niya,,,,,,
+good bye,ekosi mâka ,,,,,,
+goodbye,ekosi mâka ,,,,,,
+work together,mâmawi-isîhcikêwin,,,,,,
+working together,mâmawi-isîhcikêwin,,,,,,
+work togeth,mâmawi-isîhcikêwin,,,,,,
+Cree,nêhiyaw,,,,,,
+Edmonton,amiskwaciy-wâskahikan,,,,,,
+edmonton,amiskwaciy-wâskahikan,,,,,,
+beaver,amisk,,,,,,
+hill,waciy,,,,,,
+house,wâskahikan,,,,,,
+beaver hill,natimîwiyiniwak,amiskwacîwiyiniwak,,,,,
+beaver hills,natimîwiyiniwak,amiskwacîwiyiniwak,,,,,
+Calgary,otôskwanihk,,,,,,
+calgary,otôskwanihk,,,,,,
+cat,minos,,,,,,
+car,pôsîw,pôsiy,,,,,
+rabbit,wâpos,,,,,,
+water,nipiy,,,,,,
+telling a story,âcimêw,,,,,,
+on water,nipîhk,,,,,,
+hello,tânisi,,,,,,
+difficult thing,âyiman,,,,,,
+many snakes,kinêpikoskâw,,,,,,
+dances,nîmiw,,,,,,
+i see you,kinêpikoskâw,,,,,,
```

Comment:
```suggestion
i see you,kiwâpamitin,,,,,,
```

_kinêpikoskâw_, or _kinêpik_ ("snake) + _-iskâ_ ("an abundance of") means THERE ARE MANY SNAKES.

Reply:
Oops, yes, you tried to tell me about this before and I didn’t catch on to what you were referring to. But I do see it and the copy-paste problem now!